CHANGELOG.md

```diff
@@ -5,9 +5,9 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Update `mkdir` to take `Boolean` arg for `recursive` option (#53, #55, #58 by @JordanMartinez)
```

Comment:
Hm... This isn't entirely correct as it was `mkdir'` that was updated now. I'll need to submit a new PR.